activesupport/test/encrypted_configuration_test.rb

```diff
@@ -42,6 +42,16 @@ class EncryptedConfigurationTest < ActiveSupport::TestCase
     assert @credentials.something[:good]
   end
 
+  test "reading example configuration" do
+    require "rails/generators"
+    require "rails/generators/rails/encrypted_file/encrypted_file_generator"
```

Comment:
It is not good to rely on railties for Active Support test. 
The thing want to test here is that can correctly read encrypted files with only comments, so let's create a file with only comments in test.

Reply:
I thought it might be a good idea to ensure that the `EncryptedConfiguration` is able to read the file created by its own railties file generator.

I will change this to write a comment-only file with an `EncryptedFile`. Using an `EncryptedConfiguration` to write such a comment-only file is not possible.